runtime/gc_base/GCExtensions.hpp

```diff
@@ -182,8 +190,8 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	};
 	JitStringDeDupPolicy stringDedupPolicy;
 
-	IDATA _asyncCallbackKey;
-	IDATA _TLHAsyncCallbackKey;
+	intptr_t _asyncCallbackKey;
```

Comment:
/**< the key for async callback used in Concurrent Marking for threads to scan their own stacks */